src/react-apps/applications/service-development/src/features/iFrame/RedirectComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+export interface IRedirectComponentProvidedProps {
+  redirectUrl: string;
+  classes: any;
+}
+
+export class RedirectComponent extends
```

Comment:
How about them React Hooks? ;)

Reply:
Don't we need to upgrade react? This is only a temporarily solution so did it the way I know how to 🐑

Reply:
Yes we have... resolving.